service/gcs/bridge/bridge.go

```diff
@@ -512,34 +489,31 @@ func (b *Bridge) execProcess(w ResponseWriter, r *Request) {
 	}
 
 	if err != nil {
-		w.Error(request.ActivityID, err)
-		return
+		return nil, err
 	}
 
-	response := &prot.ContainerExecuteProcessResponse{
+	return &prot.ContainerExecuteProcessResponse{
 		MessageResponseBase: &prot.MessageResponseBase{
 			ActivityID: request.ActivityID,
```

Comment:
unecessary

Reply:
In general we should remove all references to ActivityID except at the dispatch point.